carousel/core/calculations.py

```diff
@@ -171,6 +180,15 @@ def __init__(self):
         else:
             #: parameter file
             self.param_file = None
+        # XXX: Hack to get PR#68 done
+        if not self.parameters:
```

Comment:
Ok I think I understand something: the meta parameters like 'frequency' will always be present in the registry, but they will not necessarily apply to all parameters.